docs/how-to-guides/environment-setup-and-maintenance/isis-data-area.md

```diff
@@ -0,0 +1,185 @@
+# Setting Up the ISIS Data Area
+
+Many ISIS apps need extra data to carry out their functions.  This data varies depending on the mission, and may be **quite large**, so it is **not included** with the ISIS binaries. It resides in the **ISIS Data Area**.
+
+??? info "More Info on the ISIS Data Area"
+
+    ### Structure
+
+    Each mission supported by ISIS has a directory with mission-specific processing data (like flat files and SPICE kernels).  There is also a base data area with shared data, and a test data area for ISIS testing and development.
+
+    ### Hosting
+
+    The ISIS Data Area is hosted on a combination of AWS S3 buckets and public http servers (NAIF, Jaxa, ESA), not through conda like ISIS binaries.  The `downloadIsisData` script facilitates downloading ISIS data from various sources.
+
+    ### Versions
+
+    For ISIS 4.1.0 and above, some apps require the `base` and/or mission-specific data areas, while other apps may run without them.
+
+    Apps from ISIS versions lower than 4.1.0 require the `legacybase` area.
```

Comment:
explain how to get it via `downloadIsisData`, maybe link to the legacy_base area further down and append that to mention legacy_base?

Reply:
Added some links from here to the lower section.  (I don't think more is needed, since this upper section is collapsed by default.)